TestEVStation.cpp

```diff
@@ -0,0 +1,49 @@
+#include "EVStation.h"
+void testswapValues()
+{
+    int testValue1 = 10, testValue2 = 20;
+    swapValues(&testValue1, &testValue2);
+    assert((testValue1==20) && (testValue2==10));
+}
+
+void testpartition()
+{
+    int testArray[] = {30, 10, 20};
+    int testArraySize = sizeof(testArray)/sizeof(testArray[0]);
+    partition(testArray, 0, testArraySize-1);
+    assert((testArray[0]==10) && (testArray[1]==20));
+}
+
+void testsortRange()
```

Comment:
Variations in test data is needed. Test cases can b enhanced to include different combinations of test data.